ui/design/src/components/ReadOnlyEditor/image-overlay.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { ModalContainer } from '../SignInModal/fullscreen-modal-container';
+import { Portal } from '../Editor/helpers';
+import Icon from '../Icon';
+import { StyledCloseDiv } from '../Editor/styled-editor-box';
+import styled from 'styled-components';
```

Comment:
Codacy has a fix for the issue: [Replace `styled·from·'styled-components` with `Icon·from·'../Icon';⏎import·{·ModalContainer·}·from·'../SignInModal/fullscreen-modal-container`](https://app.codacy.com/gh/AKASHAorg/akasha-world-framework/pullRequest?prid=8098754)
```suggestion
import Icon from '../Icon';
```